src/theme/components/checkbox.ts

```diff
@@ -0,0 +1,116 @@
+// @see https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/components/checkbox.ts
+// For avail theme override options.
+
+// Style object for the Checkbox's visual icon.
+const baseStyleIcon = {
+  transitionProperty: "transform",
+  transitionDuration: "normal",
+};
+
+// Style object for the Checkbox's "control."
+const baseStyleControl = {
+  //w: "100%",
+  border: "2px solid",
+  borderRadius: "sm",
+  borderColor: "ui.black",
+  color: "ui.white",
+  outline: "none",
+  transitionProperty: "box-shadow",
+  transitionDuration: "normal",
+
+  _checked: {
+    bg: "ui.white",
+    borderColor: "ui.focus",
+    color: "ui.white",
+
+    _hover: {
+      bg: "ui.white",
+    },
+
+    _disabled: {
+      bg: "ui.white",
+      borderColor: "ui.gray.medium",
+      color: "ui.gray.medium",
+    },
+  },
+
+  _indeterminate: {
+    bg: "ui.white",
+    borderColor: "ui.gray.medium",
+  },
+
+  _disabled: {
+    bg: "ui.white",
+    borderColor: "ui.gray.medium",
+  },
+
+  _focus: {
```

Comment:
I am seeing the default Chakra focus state when I preview `Checkbox` in Storybook.  Please make sure the focus state matches the DS focus state.

Reply:
- Re-worked the styles to match the DS focus state.